internal/common/baseclient/iigo.go

```diff
@@ -69,9 +69,23 @@ func (c *BaseClient) GetTaxContribution() shared.Resources {
 	}
 
 	if newTaxDecision.TaxDecided {
-		// gotVariable := newTaxDecision.ExpectedTax
-		// gotRule := newTaxDecision.TaxRule
-		return newTaxDecision.TaxAmount
+		clientContribution := float64(newTaxDecision.TaxAmount)
```

Comment:
Same as before, what if it doesn't go through the for loop? TaxAmount won't be set

Reply:
but this enters the if statement only if the for loop has changed the `newAllocationDecision`

Reply:
Hmm... Fair point. I would in any case initialise them, but as you say, it's not crucial